src/components/generic/Button.tsx

```diff
@@ -14,21 +14,37 @@ interface buttonProps {
 
 const MasterContainer = ({ title, onButtonPress, maxAnswers, answered, finished }: buttonProps) => {
   const getCurrentColor = () => {
-    if (finished) return COLORS.green;
+    if (finished) return COLORS.white;
     if (!maxAnswers || !answered) return COLORS.white;
     if (answered < maxAnswers) return COLORS.orange;
     return COLORS.green;
   };
 
+  const getButtonTextWidth = () => {
+    if (finished) return { width: '65%' };
```

Comment:
Evt kan je hier een oneliner van maken: `return finished ?  { width: '65%' } : { width: '75%' };`, ziet er misschien iets netter uit

Reply:
naast dat dit is gedaan approve ik het

Reply:
naast dat dit is gedaan approve ik het